docs/dev/RELEASE.md

```diff
@@ -98,64 +107,36 @@ with git commands and concepts is helpful.
    Running `which brew` will tell you if it is. See
    <https://docs.brew.sh/Installation> if not.
 
-1. Update brew: `brew update`
-
-2. Find the formula: `cd $(brew --repo)/Library/Taps/homebrew/homebrew-core/Formula`
-
-3. Prepare the repository:
-
-        git checkout master
-        git pull origin
-        git checkout -b <some new branch name>
-
-   The branch name is needed in various places below
-
-4. Edit the formula (e.g., `vi dafny.rb`). For a normal release change,
-   all that is needed is to change the name of the archive file for the
-   release and its SHA.
-
-   a) Change the line near the top of the file that looks like
+1. Create a GitHub personal access token (if you don't have one handy),
+   and put it in an environment variable:
 
-          url "https://github.com/dafny-lang/dafny/archive/v2.3.0.tar.gz"
-
-      to hold the actual release number (this is the url for the source
-      asset; you can see the actual name on the Releases page for the
-      latest build).
-   b) Save the file
-   c) Run `brew reinstall dafny`.
-   d) The command will complain that the SHA does not match. Copy the
-      correct SHA, reopen the file and paste the new SHA into the `sha`
-      line after the `url` line.
-   e) Bump up the revision number (by one) on the succeeding line.
-   f) Save the file
-   g) Check that you have not introduced any syntax errors by running
-      `brew reinstall dafny` again. If you totally mess up the file,
-      you can do `git checkout dafny.rb`.
+   ```
+   export HOMEBREW_GITHUB_API_TOKEN=your_token_here
+   ```
 
-5. Create a pull request following the instructions here:
+2. Create a pull request following the instructions here:
 
     <https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request>
 
-6. Expect comments from the reviewers. If changes are needed, do 4-6
-   again. Eventually the reviewers will accept and merge the PR.
+   These instructions currently involve the following command:
 
-7. Then bring your Homebrew installation back to normal:
+   ```
+  brew bump-formula-pr \
+    --url <source .tar.gz for the release> \
+    --sha256 <sha256 of the source .tar.gz for the release>
+   ```
 
-        git checkout master
-        git pull origin master
+3. Expect comments from the reviewers. If changes are needed, do 4-6
+   again. Eventually the reviewers will accept and merge the PR.
 
-8. Test the installation by running
+4. Test the installation by running
 
         brew reinstall dafny
 
    and then execute `dafny` on a sample file to see if it has the
    correct version number. Even better is to try this step on a
    different machine than the one on which the `dafny.rb` file was edited
 
-9. If everything works you can, at your leisure do
-
-        git branch -d <the branch name>
-
 ## Updating and releasing a new version of VSCode plugin
```

Comment:
I created the release process of the VSCode plugin here because I did not know it was part of the Dafny release process
https://github.com/dafny-lang/ide-vscode/blob/master/CONTRIBUTING.md

Nothing to do right now, but we should consider putting the information at only one place (probably the VSCode repository)

Reply:
Ah, good to know! I think it might be worth replacing that whole section with a link to the doc in the `ide-vscode` repo.